src/cs50/__init__.py

```diff
@@ -1,34 +1,48 @@
+import os
 import sys
 
-from .cs50 import eprint, get_char, get_float, get_int, get_string
 try:
-    from .cs50 import get_long
-except:
-    pass
 
-from . import flask
+    # Save student's sys.path
+    path = sys.path[:]
 
+    # In case student has files that shadow packages
+    sys.path = [p for p in sys.path if p not in ("", os.getcwd())]
 
-class CustomImporter(object):
-    """
-    Import cs50.SQL lazily so that rest of library can be used without SQLAlchemy installed.
+    # Import cs50_*
+    from .cs50 import eprint, get_char, get_float, get_int, get_string
+    try:
+        from .cs50 import get_long
+    except:
```

Comment:
Should probably be `except Exception:` unless we want to catch `SystemExit` et al.

Reply:
TIL